### PR TITLE
fix(widget-recents): re-add missing event fields

### PR DESCRIPTION
**Pull request by bzang**
At some point during the recents list changes a few weeks back the spaces list reference got switched to raw spaces and the events were missing the fields needed to create full events. I added those back in and put in some event tests in the journey tests to at least check fields are not empty.

### CODE REVIEW

File: test/journeys/lib/events.js [hunk 1]
```diff
@@ -29,3 +29,19 @@ export function getEventLog(myBrowser) {
...
+/**
+ * Searches events object for matching event name
+ *
+ * @export
+ * @param {object} options
+ * @param {array} options.events
+ * @param {string} options.eventName
+ * @returns {(array|boolean)}
+ */
+export function findEventName({events, eventName}) {
```
Comment (adamweeks) [inline test/journeys/lib/events.js]: Nice little helper!

File: test/journeys/lib/events.js [hunk 2]
```diff
@@ -29,3 +29,19 @@ export function getEventLog(myBrowser) {
...
 }
+
+/**
+ * Searches events object for matching event name
+ *
+ * @export
+ * @param {object} options
+ * @param {array} options.events
+ * @param {string} options.eventName
+ * @returns {(array|boolean)}
```
Comment (adamweeks) [inline test/journeys/lib/events.js]: This should always return an array, correct? When would it return boolean?

Comment (bzang) [reply test/journeys/lib/events.js]: derp, forgot to update the jsdocs after I made some changes.

**Comment by bzang** [comment]
@adamweeks, ready for another review

**Pull request merged**